Makefile

```diff
@@ -404,3 +404,11 @@ integration-tests: all testrunner
 #     integration-tests
 #
 	RUNNER_ARGS="-parallel" ./test/run-integration-tests.sh
+
+netlify-site-preview:
+	@mkdir -p site-build
```

Comment:
I think a comment regarding dependencies/where it is intended to run/etc. would indeed be useful as it doesn't seem obvious 👍 
I would still add an entry for `site-build` to `.gitignore` as we don't want to have this checked in by mistake.